int32 yourcase
float64 yourballdt
